src/libmongoc/tests/unified/entity-map.c

```diff
@@ -1236,6 +1239,10 @@ entity_client_encryption_new (entity_map_t *entity_map, bson_t *bson, bson_error
          mongoc_client_encryption_opts_set_keyvault_namespace (ce_opts, db, coll);
       }
 
+      if (key_expiration && *key_expiration >= 0) {
```

Comment:
```suggestion
      if (key_expiration) {
         BSON_ASSERT (*key_expiration >= 0);
```

Suggest erroring, rather than ignoring, if test input includes unexpected negative key expiration.